src/gmail.js

```diff
@@ -409,7 +409,7 @@ var Gmail = function(localJQuery) {
 
     api.get.storage_info = function() {
         var dom = $("[role=contentinfo] a").first();
-        var matches = dom.text().trim().match(/(\d+[.,]\d+) GB of (\d+) GB\s+\((\d+)%\)/);
+        var matches = dom.text().trim().match(/(\d+[.,]\d+) GB of (\d+) GB\s+\((\d+)\)/);
```

Comment:
This regex will still only work for english locales, and wont be applicable to non-english users.

Please see earlier feedback on this point.